picard/util/tags.py

```diff
@@ -165,653 +343,1119 @@ def names(self, selector=None):
+        see_also=('primaryreleasetype', 'secondaryreleasetype'),
     ),
     TagVar(
         'remixer',
         shortdesc=N_('Remixer'),
+        longdesc=N_('The names of the remixer engineers associated with the track.'),
     ),
     TagVar(
         'replaygain_album_gain',
         shortdesc=N_('ReplayGain Album Gain'),
+        longdesc=N_('Album gain setting resulting from ReplayGain analysis of the album.'),
         is_calculated=True,
+        is_from_mb=False,
     ),
     TagVar(
         'replaygain_album_peak',
         shortdesc=N_('ReplayGain Album Peak'),
+        longdesc=N_('Album peak setting resulting from ReplayGain analysis of the album.'),
         is_calculated=True,
+        is_from_mb=False,
     ),
     TagVar(
         'replaygain_album_range',
         shortdesc=N_('ReplayGain Album Range'),
+        longdesc=N_('Album range setting resulting from ReplayGain analysis of the album.'),
         is_calculated=True,
+        is_from_mb=False,
     ),
     TagVar(
         'replaygain_reference_loudness',
         shortdesc=N_('ReplayGain Reference Loudness'),
+        longdesc=N_('Album reference loudness setting resulting from ReplayGain analysis of the album.'),
         is_calculated=True,
+        is_from_mb=False,
     ),
     TagVar(
         'replaygain_track_gain',
         shortdesc=N_('ReplayGain Track Gain'),
+        longdesc=N_('Track gain setting resulting from ReplayGain analysis of the track.'),
         is_calculated=True,
+        is_from_mb=False,
     ),
     TagVar(
         'replaygain_track_peak',
         shortdesc=N_('ReplayGain Track Peak'),
+        longdesc=N_('Track peak setting resulting from ReplayGain analysis of the track.'),
         is_calculated=True,
+        is_from_mb=False,
     ),
     TagVar(
         'replaygain_track_range',
         shortdesc=N_('ReplayGain Track Range'),
+        longdesc=N_('Track range setting resulting from ReplayGain analysis of the track.'),
         is_calculated=True,
+        is_from_mb=False,
     ),
     TagVar(
         'sample_rate',
-        shortdesc=N_('File sample rate'),
+        shortdesc=N_('Sample Rate'),
+        longdesc=N_('The sample rate of the audio file.'),
         is_file_info=True,
         is_hidden=True,
         is_preserved=True,
         is_tag=False,
+        is_from_mb=False,
+    ),
+    TagVar(
+        'script',
+        shortdesc=N_('Script'),
+        longdesc=N_(
+            "The script used to write the release's track list. The possible values are taken from the "
+            'ISO 15924 standard.'
+        ),
     ),
     TagVar(
         'secondaryreleasetype',
-        shortdesc=N_('FIXME:secondaryreleasetype'),
+        shortdesc=N_('Secondary Release Type'),
+        longdesc=N_(
+            'Zero or more secondary types (i.e.: *audiobook*, *compilation*, *dj-mix*, *interview*, '
+            '*live*, *mixtape/street*, *remix*, *soundtrack*, or *spokenword*) for the release group.'
+        ),
```

Comment:
A good candidate for a link to MB doc about secondary types (https://musicbrainz.org/doc/Release_Group/Type)